Sources/Controllers/Panels/OAOptionsPanelBlackViewController.m

```diff
@@ -94,6 +94,15 @@ - (void) viewWillAppear:(BOOL)animated
                                                         andObserve:[OsmAndApp instance].data.weatherChangeObservable];
 }
 
+- (void)dealloc
```

Comment:
Why detach in dealloc when was attached in viewWillAppear? Use viewWillDisappear in this case.